prow/scripts/build.sh

```diff
@@ -42,6 +42,13 @@ if [[ "${BUILD_TYPE}" == "pr" ]]; then
 elif [[ "${BUILD_TYPE}" == "master" ]]; then
     make -C "${SOURCES_DIR}" ci-master
 elif [[ "${BUILD_TYPE}" == "release" ]]; then
+    NEXT_RELEASE=$(cat "${SCRIPT_DIR}/../RELEASE_VERSION")
```

Comment:
can you also add a similar check to github-release.sh to avoid accidentally overriding release?

Reply:
done